telecine/build.gradle

```diff
@@ -72,14 +72,14 @@ android {
 }
 
 dependencies {
-  compile 'com.android.support:support-annotations:23.2.1'
+  compile 'com.android.support:support-annotations:23.4.0'
```

Comment:
You can make one PR, but I would prefer separate commits for each. Sometimes I'm not all that strict about it, but it makes doing things like `git bisect` a lot easier should there be a problem in the future.